org.zenframework.z8.server/src/main/java/org/zenframework/z8/server/base/table/system/UserRoles.java

```diff
@@ -54,6 +59,8 @@ public CLASS(IObject container) {
 			setJavaClass(UserRoles.class);
 			setName(TableName);
 			setDisplayName(displayNames.Title);
+			Optional.ofNullable(apiAttrs.Title)
+					.ifPresent(attrVal -> setAttribute("APIDescription", attrVal));

```

Comment:
Write without lambda (->), declare APIDescription constant in org.zenframework.z8.server.json.Json

Reply:
what's wrong with lambda?
make something like that?
`if (apiAttrs.Title != null) {
    setAttribute(....)
}`